app/common_functions/check_for_id.py

```diff
@@ -0,0 +1,15 @@
+from flask import make_response, abort
+
+def valid_int(number, parameter_type):
+    try:
+        int(number)
+    except:
+        abort(make_response({"error": f"{parameter_type} must be an int"}, 400))
+
+def get_id(id, model, str_repr):
```

Comment:
As Makhabat and I discussed in our one on one, you may be able to use `model.__name__` or something similar for the string representation.